app/javascript/packs/document-capture.tsx

```diff
@@ -78,31 +67,12 @@ const trackEvent: typeof baseTrackEvent = (event, payload) => {
   });
 };
 
-(async () => {
-  const backgroundUploadURLs = getBackgroundUploadURLs();
-  const isAsyncForm = Object.keys(backgroundUploadURLs).length > 0;
-
+(() => {
```

Comment:
We can remove this IIFE `(() => {});` block. (Downside being it would make a lot of diff noise with the whitespace de-indenting)

Reply:
Removed. Checked in as the only change in a commit, so that's at least a little help with sorting through the noise.